src/main/webapp/WEB-INF/country.jsp

```diff
@@ -35,15 +56,32 @@
           <div>
     <%  Iterator iter = categories.iterator();
         while (iter.hasNext()) {
-          String category = (String) iter.next();    %>
-          <a href="/country/<%= countryCode %>/c/<%= category %>"><%= category %></a>
+          String categoryList = (String) iter.next();    %>
+          <a href="/country/<%= countryCode %>/c/<%= categoryList %>"><%= categoryList %></a>
     <%  }   %>
   </div>
         </div>
       </div>
     </div>
     <h1 id="page-title"><%= countryName %></h1>
 
+    <% if (currentUser != null) { %>
+    <form id="message-form" action="" method="POST" class>
+    Enter a new message:
+    <br/>
+    <textarea name="text" placeholder="Enter a message" id="message-input"></textarea>
+      <select id="myDropdown" onchange="updateMessageCategory()">
+        <!-- <option value="" selected disabled hidden>Select Category</option> -->
```

Comment:
Is this debugging stuff?

Reply:
It was a placeholder until the user selected a category value, but its tricky to deal with the user clicking submit without changing the category. So I just made the default category the General one.